website/docs/r/cluster.html.markdown

```diff
@@ -348,6 +348,13 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri` - Base connection string for the cluster. Atlas only displays this field after the cluster is operational, not while it builds the cluster.
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
+* `connection_strings` - Set of connection strings that your applications use to connect to this cluster.
+    - `connection_strings.standard` -  `mongodb://` A required prefix to identify that this is a string in the standard connection format.
+    - `connection_strings.standard_srv` - The `mongodb+srv` protocol tells the driver to look up the seed list of hosts in DNS.
+    - `connection_strings.aws_private_link` - Connection strings for each interface VPC endpoint you configured to connect to this cluster.
```

Comment:
- `connection_strings.aws_private_link` - [Private-endpoint-aware](https://docs.atlas.mongodb.com/security-private-endpoint/#private-endpoint-connection-strings) mongodb://connection strings for each interface VPC endpoint you configured to connect to this cluster. Returned only if you created a AWS PrivateLink connection to this cluster.